packages/medusa/src/api/routes/admin/users/index.ts

```diff
@@ -30,11 +31,31 @@ export default (app) => {
 
   route.delete("/:user_id", middlewares.wrap(require("./delete-user").default))
 
-  route.get("/", middlewares.wrap(require("./list-users").default))
+  route.get(
+    "/",
+    transformQuery(AdminGetUsersParams, {
+      defaultFields: defaultAdminUserFields,
+      isList: true,
+    }),
+    middlewares.wrap(require("./list-users").default)
+  )
 
   return app
 }
 
+export const defaultAdminUserFields: (keyof User)[] = [
+  "id",
+  "email",
+  "first_name",
+  "last_name",
+  "role",
+  "api_token",
```

Comment:
thought(non-blocking): maybe we don't want this to be exposed to everyone, but only the logged in user? Don't know how it's been in the past though

Reply:
@srindom I thought so to, but the integration tests failed when I initially made it so the listAndCount endpoint didn't return it. I am down to removing it, as I also think it is weird that this very "private" thing is available to all users, but it would be a breaking change in some sense.